src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
```

Comment:
box-sizing should be the same for all elements, move it to `*` selector in normalize